### PR TITLE
Hungama alternate source

### PR DESCRIPTION
**Pull request by Dum4G**
OG source drops connection after a while. I'm making a tvg-logo difference intentional here

### CODE REVIEW

File: channels/in.m3u [hunk 1]
```diff
@@ -728,6 +728,8 @@ https://news18kannada-lh.akamaihd.net/i/n18kannada_1@372918/index_4_av-p.m3u8?re
 #EXTINF:-1 tvg-id="PublicMusic.in" tvg-name="Public Music" tvg-country="IN" tvg-language="Kannada" tvg-logo="https://upload.wikimedia.org/wikipedia/en/3/35/Public_Music_Kannada_Channel_Logo.jpg" group-title="Music",ಪಬ್ಲಿಕ್ ಸಂಗೀತ Public Music (576p)
 http://103.199.161.254/Content/publicmusic/Live/Channel(PublicMusic)/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/ctjMaOW.png" group-title="Kids",Hungama TV
+http://70.39.83.58:8278/hungamahindikdin/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/ctjMaOW.png" group-title="Kids",Hungama TV
```
Comment (Dum4G) [inline channels/in.m3u:732]:
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/9/9c/Hungama_TV.svg/220px-Hungama_TV.svg.png" group-title="Kids",Hungama TV
```

**Comment by freearhey** [comment]
So maybe just replace the other link with a new one if it's bad?

**Comment by Dum4G** [comment]
We can't guarantee total availability of every source we have. And we can't predict which one of them will last for longer period. I think this time a choice option is needed

File: channels/in.m3u [hunk 2]
```diff
@@ -728,6 +728,8 @@ https://news18kannada-lh.akamaihd.net/i/n18kannada_1@372918/index_4_av-p.m3u8?re
 #EXTINF:-1 tvg-id="PublicMusic.in" tvg-name="Public Music" tvg-country="IN" tvg-language="Kannada" tvg-logo="https://upload.wikimedia.org/wikipedia/en/3/35/Public_Music_Kannada_Channel_Logo.jpg" group-title="Music",ಪಬ್ಲಿಕ್ ಸಂಗೀತ Public Music (576p)
 http://103.199.161.254/Content/publicmusic/Live/Channel(PublicMusic)/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/ctjMaOW.png" group-title="Kids",Hungama TV
+http://70.39.83.58:8278/hungamahindikdin/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/9/9c/Hungama_TV.svg/220px-Hungama_TV.svg.png" group-title="Kids",Hungama TV
```
Comment (freearhey) [inline channels/in.m3u:732]: Wouldn't it be better to just add the labeling to the header then?
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/ctjMaOW.png" group-title="Kids",Hungama TV (Alternative)
```

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**